core/src/test/java/org/apache/accumulo/core/client/ClientConfigurationTest.java

```diff
@@ -23,12 +23,20 @@
 import java.io.File;
 import java.io.IOException;
 
-import org.apache.accumulo.core.client.ClientConfiguration.ClientProperty;
 import org.junit.Test;
 
-@Deprecated
+@SuppressWarnings("deprecation")
 public class ClientConfigurationTest {
 
+  @SuppressWarnings("deprecation")
```

Comment:
These SuppressWarnings are unnecessary, since they are already suppressed at the top of the class.